frontend/src/libs/packages/store/middlewares/handle-unauthorized/handle-unauthorized.middleware.ts

```diff
@@ -0,0 +1,26 @@
+import { type AnyAction, type Middleware } from '@reduxjs/toolkit';
+import { isRejected } from '@reduxjs/toolkit';
+
+import { type AppDispatch } from '#libs/packages/store/libs/types/types.js';
+import { actions as authActions } from '#slices/auth/auth.js';
+
+import { ExceptionMessage } from '../../libs/enums/enums.js';
+
+const handleUnauthorized: Middleware<unknown, unknown, AppDispatch> = ({
+  dispatch,
+}) => {
+  return (next) => {
+    return (action: AnyAction) => {
+      if (isRejected(action)) {
+        const isUnauthorized =
```

Comment:
compare them by error name, not exception message

Reply:
@Liza-Veis  as far as I tried to test it with other errors - the error.name is always 'Error'

Reply:
Just pass name to custom error

Reply:
Done!